appveyor.yml

```diff
@@ -51,9 +51,9 @@
   init:
   - cd "C:\projects\reactiveui"
   install: 
-  - cmd: nuget install SignClient -Version 0.7.0 -SolutionDir %APPVEYOR_BUILD_FOLDER% -Verbosity quiet -ExcludeVersion
+  - cmd: nuget install SignClient -Version 0.9.0 -SolutionDir %APPVEYOR_BUILD_FOLDER% -Verbosity quiet -ExcludeVersion
   build_script:
-  - ./build.cmd
+  - ./build.cmd && exit 0 
```

Comment:
Little bit concerned about `&& exit 0` @onovotny. Why? Wouldn't this make the build green, even if there was a failure?

Reply:
Was an attempt to get the scripts continue and upload artifacts. Can be removed. There's still an outstanding issue where if there's an error within the Cake build, the *.binlog artifacts are not pulled. It may need something in Cake in the Teardown block if running in AppVeyor. Made troubleshooting really hard.

Reply:
Opened up #1488 msbuild artifacts not uploading as seperate issue and PR to remove exit 0 over at #1489

Reply:
k, sorry for forgetting the `exit 0` before I squashed